src/tree/CosmosDBAccountProvider.ts

```diff
@@ -38,7 +38,8 @@ export class CosmosDBAccountProvider implements IChildProvider {
                     accountTreeItems.push(account);
                 } catch (e) {
                     const err = parseError(e);
-                    accountTreeItems.push(<IAzureTreeItem>{ id: err.toString(), label: err.toString() });
+                    //tslint:disable-next-line:no-non-null-assertion
+                    accountTreeItems.push(<IAzureTreeItem>{ label: databaseAccount!.name, description: "Invalid: " + err.message });
```

Comment:
Can you paste a screenshot?  I'm not sure the "Invalid:" will be necessary, and it will reduce space.

Reply:
![image](https://user-images.githubusercontent.com/30420973/40458675-25b007be-5eb2-11e8-9333-cf4128ff346c.png)

Reply:
Yeah, I'd suggest getting rid of "Invalid:"